src/scripts/main.js

```diff
@@ -1,3 +1,79 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const field = document.querySelector('.field');
+const fieldBody = field.tBodies[0];
```

Comment:
Ensure that the `field` element has at least one `tbody` element. If `field.tBodies[0]` is undefined, it will cause a runtime error. Consider adding a check to ensure `field.tBodies[0]` exists before accessing it.